src/renderers/testing/ReactTestMount.js

```diff
@@ -20,6 +20,16 @@ var getHostComponentFromComposite = require('getHostComponentFromComposite');
 var instantiateReactComponent = require('instantiateReactComponent');
 var invariant = require('invariant');
 
+type TestRendererMockConfig = {
+  getMockRef: (element: ReactElement) => Object,
+};
+
+var defaultMockConfig = {
```

Comment:
Let's also rename "mock config" to "test options" everywhere. It's longer but more explicitly and makes it clear we can add more stuff to it if we want to that's unrelated to mocking.

Inside test renderer we can just call it `options`. Outside, `testOptions` and `getTestOptions` seem fine.